erpnext/accounts/doctype/subscription/subscription.py

```diff
@@ -280,7 +280,7 @@ def create_invoice(self, prorate):
 
 		if self.additional_discount_percentage or self.additional_discount_amount:
 			discount_on = self.apply_additional_discount
-			invoice.apply_additional_discount = discount_on if discount_on else 'Grand Total'
+			invoice.apply_discount_on = discount_on if discount_on else 'Grand Total'
```

Comment:
Why this change?

Reply:
there's no field called `apply_additional_discount` in Sales Invoice. The concerned field is named `apply_discount_on`. 

<img width="1280" alt="Screenshot 2020-01-27 at 3 48 33 PM" src="https://user-images.githubusercontent.com/6195660/73166797-79ea1700-40ee-11ea-8cb7-421e4b9f087c.png">